src/data/audio-service.js

```diff
@@ -39,7 +39,7 @@ const AudioService = {
 
     return dev
       ? Promise.resolve({ data: sampleAudioResponses.detail })
-      : ApiService.get('audio', params.id)
+      : ApiService.get('audio/', params.id)
```

Comment:
Won't this result in double slashes? The `get` function in the api service already adds a slash after the resource

Reply:
@dhruvkb automating it via some axios adapter would prevent confusions like this ^. That's the main benefit to me, then we never have to think about it again. I see it the same as when we adopt an ESLint rule, just in this case we can fix the problem easily at runtime.